spec/arrays.dd

```diff
@@ -165,7 +165,10 @@ $(H2 $(LNAME2 slicing, Slicing))
 
 $(SPEC_RUNNABLE_EXAMPLE_COMPILE
```

Comment:
Let's better use `SPEC_RUNNABLE_EXAMPLE_RUN` here, s.t. the spec tester actually runs the example?